invisible-plugin-sample/README.md

```diff
@@ -0,0 +1,3 @@
+# UXP Invisible Plugin Sample
+
+This Plugin is an example of creating invisible plugin in UXP.
```

Comment:
Could you please elaborate a little bit about what an invisible plugin is here? This doesn't have to be a wall of text, but just describe in one or two sentences what the plugin does.

Reply:
@pklaschka thanks for your review. is it possible for you to update the description if i explain it here?